database/migrate/migrations/00015_transaction.sql

```diff
@@ -0,0 +1,46 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE transaction
```

Comment:
how about change the name to `pending_transaction`?

Reply:
changed in [88de67b](https://github.com/scroll-tech/scroll/pull/1076/commits/88de67b28bfa9a0183d642c1d6e44a37ffefb732)